ci/helm/values/env-values-global.yaml

```diff
@@ -0,0 +1,4 @@
+global:
+  labels:
+    dev.nuxeo.com/repository: nuxeo-explorer
+    dev.nuxeo.com/team: platform
```

Comment:
Not needed a already in the remote ci/helm/apps/base/values/env-values-global.yaml.gotmpl?